ground/src/main/java/com/google/android/ground/util/Debug.kt

```diff
@@ -15,13 +15,19 @@
  */
 package com.google.android.ground.util
 
+import com.google.android.ground.FirebaseCrashLogger
 import timber.log.Timber
 
 object Debug {
   fun logLifecycleEvent(instance: Any) {
     val stackTrace = Thread.currentThread().stackTrace
     val callingMethod = stackTrace[3].methodName + "()"
-    Timber.tag(instance.javaClass.simpleName).v("Lifecycle event: $callingMethod")
+    val className = instance.javaClass.simpleName
+    Timber.tag(className).v("Lifecycle event: $callingMethod")
+
+    if (callingMethod == "onResume()") {
+      FirebaseCrashLogger().setScreenName(className)
```

Comment:
What do you think about remaining to something like fragmentClass so that what this string refers to is more self documenting?

Also, are we sure the fragment's is always called last? Are there any other lifecycle objects which might be called after? Since there's no compile check for this, we may want to check the type of the instance to make sure we got the right one.